react-redux-app/src/componets/Sort/Sort.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './Sort.css'
+import arrow from '../../img/arrow.svg'
+import searce from '../../img/searce.png'
+import { sortDate } from '../../store/listSlice'
+import { useDispatch } from 'react-redux'
+
+const Sort = () => {
+    const dispatch = useDispatch(sortDate)
```

Comment:
it is not right, you call the hook already with some function, you must pass function into dispatch only when call it
const dispatch = useDispatch()